apps/super-app/src/common/contexts/ThemeProvider.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { ThemeProvider as MuThemeProvider } from '@mui/material/styles';
```

Comment:
nit: have a feeling this is intended to be `MuiThemeProvider` instead?

Reply:
Oh you're right.